src/components/FormattedNumber/FormattedNumber.jsx

```diff
@@ -29,13 +46,26 @@ const FormattedNumber = forwardRef(
     }, [value, decimalPrecision, local, compact]);
 
     if (validateValue(value)) {
-      return <span className={`${baseClassName}__place-holder`}>{emptyPlaceHolder}</span>;
+      return (
+        <span
+          className={cx(`${CSS_BASE_CLASS}__place-holder`)}
+          id={id}
+          data-testid={dataTestId || getTestId(ELEMENT_TYPES.FORMATTED_NUMBER, id)}
+        >
+          {emptyPlaceHolder}
+        </span>
+      );
     }
 
     return (
-      <div ref={ref} className={classNames(className, baseClassName)}>
+      <div
+        ref={ref}
+        className={cx(className, styles.formattedNumber, CSS_BASE_CLASS)}
+        id={id}
+        data-testid={dataTestId || getTestId(ELEMENT_TYPES.FORMATTED_NUMBER, id)}
+      >
         {rtl ? renderSuffix : renderPrefix}
-        <span className={`${baseClassName}__number`}>{calculatedValue}</span>
+        <span className={cx(`${CSS_BASE_CLASS}__number`)}>{calculatedValue}</span>
```

Comment:
no css modules class here?

Reply:
That's correct. There is no such class in the stylesheet - so it's being used only for overrides